src/wp-admin/includes/media.php

```diff
@@ -3843,3 +3843,12 @@ function wp_media_attach_action( $parent_id, $action = 'attach' ) {
 		exit;
 	}
 }
+
+/**
+ * Returns the default image output mapping.
+ *
+ * @since 6.1.0
```

Comment:
This needs a `@return` doc, and if you follow my other comment below also a `@param` doc.
```suggestion
 * @since 6.1.0
 *
 * @param array $output_format The array of mime type mappings for output.
 * @return array The adjusted $output_format.
```

Reply:
adjusted